ui/imports/shared/views/chat/StatusUpdateView.qml

```diff
@@ -70,9 +70,7 @@ MouseArea {
 //            profileImage: root.profileImageSource
```

Comment:
This file is probably not used anymore, since we removed StatusUpdates, so maybe we can delete this file?

Reply:
Yeah, seemed unused, removed.